index.js

```diff
@@ -0,0 +1,10 @@
+var twitterText = require('twitter-text');
+
+module.exports = function(text, imageAttachments) {
+  var charsLeft = 140;
+
+  // Images (1 or more), use up 24 characters.
+  if (imageAttachments === true) { charsLeft -= 24; };
```

Comment:
The semicolon at the very end of this line is not necessary under any circumstance.